.gitignore

```diff
@@ -9,3 +9,5 @@ develop-eggs/
 tags
 .tox
 parts/
+.coverage
```

Comment:
Given `parallel = true`, maybe also ignore `.coverage.*`?  OTOH those are deleted when you (or tox) run `coverage combine`, so maybe we don't care.

Reply:
Seems reasonable enough. If a subprocess crashes or fails then tox doesn't run the next command, right? So failing tests would leave these lying around.